api/app/signals/apps/signals/models/status_message_template.py

```diff
@@ -44,7 +45,7 @@ def save(self, *args, **kwargs):
         self.full_clean()
         qs = StatusMessageTemplate.objects.filter(category_id=self.category_id, state=self.state)
 
-        if self.pk is None and qs.count() >= MAX_INSTANCES:
+        if self.pk is None and qs.count() >= settings.STATUS_MESSAGE_TEMPLATE_MAX_INSTANCES:
             msg = f'Only {MAX_INSTANCES} StatusMessageTemplate instances allowed per Category/State combination'
```

Comment:
I don't think we need both `MAX_INSTANCES` and `settings.STATUS_MESSAGE_TEMPLATE_MAX_INSTANCES`.